From private repo

v-alje opened the pull request.

azurecla commented:
Hi __@hillaryc__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla.azure.com.
    </span>

TTYL, AZPRBOT;

deneha commented:
Author is MSFT employee - CLA not required

@hillaryc Thank you for this contribution.  New articles should be submitted via the private repo - azure-content-pr.  Would you please submit this article there instead and close this request?

@sethmanheim Looping you in on this new content request for Service Bus.

Commit:
rajat41

Commit:
rajat41

Ja-Dunn committed:
edit pass: three-vs-articles

Ja-Dunn committed:
Merge branch 'master' of https://github.com/microsoft/azure-docs-pr into three-vs-articles

Ja-Dunn committed:
edit pass: three-vs-articles

Ja-Dunn committed:
Merge branch 'master' of https://github.com/microsoft/azure-docs-pr into three-vs-articles

Ja-Dunn committed:
edit pass: three-vs-articles

Ja-Dunn committed:
Merge branch 'master' of https://github.com/microsoft/azure-docs-pr into three-vs-articles

Ja-Dunn committed:
edit pass: three-vs-articles

GitHubber17 committed:
edit pass: predict-twitter-sentiment

GitHubber17 committed:
Update predict-twitter-sentiment.md

GitHubber17 committed:
edit pass: acrolinx updates

GitHubber17 committed:
edit pass: figure intros

TerryLanfear committed:
new article

TerryLanfear committed:
fix

TerryLanfear committed:
fixes per acrolinx

TerryLanfear committed:
fixes per acrolinx

TerryLanfear committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into os-config

TerryLanfear committed:
fixes

TerryLanfear committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into os-config

TerryLanfear committed:
final fixes

TerryLanfear committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into os-config

TerryLanfear committed:
TOC update

garyericson committed:
Renaming 'Cortana Intelligence Gallery' to 'Azure AI Gallery' #2

garyericson committed:
Renaming 'Cortana Intelligence Gallery' to 'Azure AI Gallery' #3

garyericson committed:
Fixed Acrolinx issues

garyericson committed:
Fixed name in include file

garyericson committed:
Updated screenshots with new Gallery name

garyericson committed:
Added borders on new images

garyericson committed:
Fixed one more image

MGoedtel committed:
updated article per CSS input

MGoedtel committed:
updated with minor readability edits

Ja-Dunn committed:
Merge branch 'master' of https://github.com/microsoft/azure-docs-pr into three-vs-articles

Ja-Dunn committed:
Merge branch 'master' of https://github.com/microsoft/azure-docs-pr into three-vs-articles

sanjeev3 committed:
Update image-moderation-quickstart-dotnet.md

sanjeev3 committed:
Update text-moderation-quickstart-dotnet.md

sanjeev3 committed:
Update image-lists-quickstart-dotnet.md

Ja-Dunn committed:
edit pass: three-vs-articles

garyericson committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into 01-04-gallery-rename-3

garyericson committed:
Replaced main screenshot, updated the way it's referenced

MGoedtel committed:
updated with additional information

garyericson committed:
Trying different table formatting

Ja-Dunn committed:
Merge branch 'master' of https://github.com/microsoft/azure-docs-pr into three-vs-articles

markgalioto committed:
changing the references to classic portal

garyericson committed:
Fixing table formatting

markgalioto committed:
improving the acrolinx scores

Ja-Dunn committed:
minor fixes

markgalioto committed:
removed mention of Classic portal

markgalioto committed:
changing formatting in intro article for Backup

JasonWHowell committed:
Update concepts-limits.md

JasonWHowell committed:
Update concepts-limits.md

mimig1 committed:
Cosmos DB: Updated graph endpoint.

Ja-Dunn committed:
Update backup-install-agent.md

typo fix

jlian committed:
Added delay disclaimer

jlian committed:
Fix typo

garyericson committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into 01-04-gallery-rename-3

cwatson-cat committed:
Removed reference to Azure Management Portal.

Ja-Dunn committed:
Merge pull request #30956 from markgalioto/ab-1-5-2018

changing the references to classic portal

garyericson committed:
Updating real article with results from test article

Ja-Dunn committed:
Merge pull request #30973 from JasonWHowell/patch-4

PostgreSQL: Update concepts-limits.md

Ja-Dunn committed:
Merge pull request #30976 from jlian/master

Added delay disclaimer

cwatson-cat committed:
Removed referenced to management portal in CDN docs

ShawnJackson committed:
Merge pull request #30616 from Ja-Dunn/three-vs-articles

edit pass: three-vs-articles

cwatson-cat committed:
Missed a reference

PRMerger18 committed:
Merge pull request #29534 from MarkusVi/rajat41

rajat41

iainfoulds committed:
Update CLI code fencing

garyericson committed:
Fixed paragraph breaks

cwatson-cat committed:
Removed reference to Azure management portal

ktoliver committed:
Merge pull request #30980 from iainfoulds/patch-1

VMSS OS upgrades

SnehaGunda committed:
Updating author name

Ja-Dunn committed:
Merge pull request #30977 from cwatson-cat/1-5-18-ClassicClnUp-Automation-MgmPortal

Removed reference to Azure Management Portal.

ktoliver committed:
Merge pull request #30897 from garyericson/01-04-gallery-rename-3

Renaming 'Cortana Intelligence Gallery' to 'Azure AI Gallery' #3

Ja-Dunn committed:
Merge pull request #30895 from garyericson/01-04-gallery-rename-2

Renaming 'Cortana Intelligence Gallery' to 'Azure AI Gallery' #2

cwatson-cat committed:
Improve acrolynx score

tdykstra committed:
acrolinx

cjgronlund committed:
small format change

TerryLanfear committed:
fixes

tdykstra committed:
acrolinx

ShawnJackson committed:
Update predict-twitter-sentiment.md

ShawnJackson committed:
Merge pull request #30621 from GitHubber17/1148960

edit pass: predict-twitter-sentiment

TerryLanfear committed:
fixed merge conflict

TerryLanfear committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into os-config

Commit:
Pri 0 - Mitigating speculative execution

Commit:
acrolinx

Commit:
toc changes

barlan committed:
pulling classic

barlan committed:
pulling last instances

Commit:
additional changes

Commit:
last sentence

tynevi committed:
Merge pull request #30995 from cynthn/CPU-2

Pri 0 - Mitigating speculative execution

sanjeev3 committed:
Create moderation-reviews-quickstart-dotnet.md

sanjeev3 committed:
Update toc.yml

sanjeev3 committed:
Add files via upload

sanjeev3 committed:
Update moderation-reviews-quickstart-dotnet.md

sanjeev3 committed:
Delete moderation-reviews-quickstart-dotnet.PNG

sanjeev3 committed:
Add files via upload

sanjeev3 committed:
Update moderation-reviews-quickstart-dotnet.md

sanjeev3 committed:
Update moderation-reviews-quickstart-dotnet.md

sanjeev3 committed:
Update moderation-reviews-quickstart-dotnet.md

sanjeev3 committed:
Update moderation-reviews-quickstart-dotnet.md

MGoedtel committed:
minor update to improve point

sanjeev3 committed:
Update moderation-reviews-quickstart-dotnet.md

sanjeev3 committed:
Update moderation-reviews-quickstart-dotnet.md

sanjeev3 committed:
Update moderation-reviews-quickstart-dotnet.md

Commit:
3:00 PM updates

Commit:
acrolinx

Commit:
time:

Commit:
Adding to CS toc

Commit:
bracket

Commit:
cs include

tynevi committed:
Merge pull request #31001 from cynthn/revision1

CPU Pri 0 - 3:00 PM updates

sanjeev3 committed:
Add files via upload

sanjeev3 committed:
Create moderation-jobs-quickstart-dotnet.md

sanjeev3 committed:
Update moderation-jobs-quickstart-dotnet.md

sanjeev3 committed:
Update moderation-jobs-quickstart-dotnet.md

sanjeev3 committed:
Update moderation-reviews-quickstart-dotnet.md

sanjeev3 committed:
Update toc.yml

sanjeev3 committed:
Update moderation-reviews-quickstart-dotnet.md

sanjeev3 committed:
Update moderation-jobs-quickstart-dotnet.md

sanjeev3 committed:
Update moderation-jobs-quickstart-dotnet.md

sanjeev3 committed:
Update moderation-reviews-quickstart-dotnet.md

Commit:
html table

cynthn committed:
Merge pull request #31005 from cynthn/revision2

CPU article formatting

TerryLanfear committed:
fixes

TerryLanfear committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into os-config

rayne-wiselman committed:
updating a few issues in overview

rayne-wiselman committed:
Update site-recovery-overview.md

sanjeev3 committed:
Update moderation-jobs-quickstart-dotnet.md

gsacavdm committed:
Fix formatting for code blocks

sanjeev3 committed:
Update moderation-jobs-quickstart-dotnet.md

sanjeev3 committed:
Update moderation-jobs-quickstart-dotnet.md

sanjeev3 committed:
Update moderation-jobs-quickstart-dotnet.md

TerryLanfear committed:
fixes

Lisaco88 committed:
Merge pull request #30921 from MGoedtel/UpdateAlertOverview152018

updated article per CSS input

rjagiewich committed:
Merge pull request #30979 from cwatson-cat/1-5-18-ClassicClnUp-CDN-MgmPortal

Removed referenced to management portal in CDN docs

Lisaco88 committed:
Merge pull request #30981 from cwatson-cat/1-5-18-ClassicClnUp-Log-MgmPortal

Removed reference to Azure management portal

Lisaco88 committed:
Merge pull request #30987 from tdykstra/patch-5

acrolinx

rjagiewich committed:
Merge pull request #30984 from SnehaGunda/TSI

Updating author name

GitHubber17 committed:
Merge pull request #30814 from TerryLanfear/os-config

new article

Lisaco88 committed:
Update data-science-for-beginners-copy-other-peoples-work-to-do-data-science.md

Removed extraneous word from H2

rjagiewich committed:
Merge pull request #30989 from tdykstra/patch-6

acrolinx

Lisaco88 committed:
Merge pull request #30988 from cjgronlund/fresh-jan4

small format change

sanjeev3 committed:
Add files via upload

arob98 committed:
update build number

arob98 committed:
known issue update

GitHubber17 committed:
Merge pull request #31008 from rayne-wiselman/rayne-january7

updating a few issues in overview

sanjeev3 committed:
Update Workflows.md

tynevi committed:
Merge pull request #31015 from arob98/010718

update build number

sanjeev3 committed:
Delete ocr-workflow-step-1.PNG

sanjeev3 committed:
Delete ocr-workflow-step-2-condition.PNG

sanjeev3 committed:
Delete ocr-workflow-step-3-action.PNG

sanjeev3 committed:
Delete ocr-workflow-step-4-test.PNG

sanjeev3 committed:
Delete ocr-workflow-step-5-review.PNG

sanjeev3 committed:
Add files via upload

sanjeev3 committed:
Add files via upload

sanjeev3 committed:
Update Workflows.md

sanjeev3 committed:
Update moderation-reviews-quickstart-dotnet.md

sanjeev3 committed:
Update try-review-api-job.md

sanjeev3 committed:
Update try-review-api-review.md

sanjeev3 committed:
Update try-review-api-workflow.md

linda33wj committed:
Remove Azure IR support for Web connector

linda33wj committed:
Add unsupported scenario for fault tolerance

linda33wj committed:
Update the links of ADLS and SQL DB cross ref on ADF

ChrisGMsft committed:
update the sample

anoobbacker committed:
Adding the https://login.microsoftonline.us/

Adding the new URL.

anoobbacker committed:
Update storsimple-8000-system-requirements.md

JYOTHIRMAISURI committed:
updates to ITSMC articles

nsoneji committed:
Added version history link and updated the VM  limits as per 2.1 release

nsoneji committed:
minor correction.

nsoneji committed:
correction 2.0 -> 2.1

rayne-wiselman committed:
redirecting file, updating UI changes

Commit:
removing the priority 500 rule from snippets per engineering

Commit:
updating date for freshness

Brenduns committed:
Version fix and formattiong fix

Commit:
Removing a low traffic, high-maintenance document and redirecting to a code example

RicksterCDN committed:
updating Oracle Linux compat with M128

Lisaco88 committed:
Merge pull request #30996 from barlanmsft/barlan0105

pulling classic

Lisaco88 committed:
Merge pull request #31013 from gsacavdm/patch-1

Fix formatting for code blocks

sivethe committed:
Change GetLastRequestStatistics section for H3 to H2

Change GetLastRequestStatistics section for H3 to H2 so that is easier to find.  We direct a lot of customer to this page as this is an extension MongoDB command specific to CosmosDB

mimig1 committed:
update

iainfoulds committed:
Update instance-metadata-service.md

iainfoulds committed:
Update instance-metadata-service.md

itechedit committed:
Merge pull request #31017 from sanjeev3/master

New quickstarts and TOC and article updates

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPublicRepo

ktoliver committed:
Merge pull request #31020 from ChrisGMsft/patch-17

update the sample

mimig1 committed:
Added note that Cosmos DB only works for SQL and Graph APIs.

Lisaco88 committed:
fixed typos

ktoliver committed:
Merge pull request #31024 from JYOTHIRMAISURI/master

updates to ITSMC articles

Lisaco88 committed:
fixed typos

v-alje committed:
Merge pull request #31039 from MicrosoftDocs/FromPublicRepo

From public repo

rmca14 committed:
Merge pull request #31022 from anoobbacker/patch-25

Adding the https://login.microsoftonline.us/

mimig1 committed:
Cosmos DB: Moved note up.

tdykstra committed:
fix fenced code block

Lisaco88 committed:
Merge pull request #31019 from linda33wj/master

Update ADF copy content

GitHubber17 committed:
Merge pull request #31029 from Blackmist/freshness

Freshness

ktoliver committed:
Merge pull request #31027 from rayne-wiselman/rayne-december8

redirecting file, updating UI changes

rmca14 committed:
Merge pull request #31026 from nsoneji/master

Added version history link and updated the VM  limits as per 2.1 release

GitHubber17 committed:
Merge pull request #31030 from Brenduns/daily-0108

Version fix and formattiong fix

GitHubber17 committed:
Merge pull request #31034 from RicksterCDN/OracleM128

updating Oracle Linux compat with M128

ktoliver committed:
Merge pull request #31031 from Blackmist/fresh18

Removing a low traffic, high-maintenance document and redirecting to …

GitHubber17 committed:
Merge pull request #31037 from sivethe/patch-2

Change GetLastRequestStatistics section for H3 to H2

GitHubber17 committed:
Merge pull request #31040 from mimig1/patch-189

Added note that Cosmos DB only works for SQL and Graph APIs.

GitHubber17 committed:
Merge pull request #31045 from mimig1/patch-190

Cosmos DB: Moved note up.

GitHubber17 committed:
Merge pull request #31046 from tdykstra/json

fix fenced code block

ktoliver committed:
Merge pull request #31038 from iainfoulds/patch-2

VM Instance Metadata Service

PRMerger14 committed:
Merge pull request #30974 from mimig1/01052018

Cosmos DB: Updated graph endpoint.
